src/apps/advanced-search-v2/components/OperatorButtons.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import clsx from "clsx";
+import { useText } from "../../../core/utils/text";
+import { Operator } from "../types";
+
+type OperatorButtonsProps = {
+  value: Operator;
+  onChange: (operator: Operator) => void;
+};
+
+const OperatorButtons: React.FC<OperatorButtonsProps> = ({
+  value,
+  onChange
+}) => {
+  const t = useText();
+
+  const operators: Array<{ value: Operator; label: string }> = [
+    { value: "and", label: t("clauseAndText") },
+    { value: "or", label: t("clauseOrText") },
+    { value: "not", label: t("clauseNotText") }
+  ];
```

Comment:
Could this be moved to a shared constant with an array of Operator options or the like.